### ISSUE TITLE
💅 svelte: variables only used as attachments trigger noUnusedVariables lint rule

### ISSUE THREAD

**Issue by machadinhos** [biomejs/biome#11171]
### Environment information

<details>

```bash

I was not able to run this command for whatever reason, I get no error, it does nothing.
I'm on windows.

```

</details>

### Rule name

noUnusedVariables 

### Playground link

https://biomejs.dev/playground/?tab=formatter&pane=Diagnostics&code=PABzAGMAcgBpAHAAdAAgAGwAYQBuAGcAPQAiAHQAcwAiAD4ACgAgACAAYwBvAG4AcwB0ACAAZgBvAG8AIAA9ACAAKABuAG8AZABlACkAIAA9AD4AIAB7AAoAIAAgACAAIABjAG8AbgBzAG8AbABlAC4AbABvAGcAKABuAG8AZABlACkAOwAKACAAIAB9ADsACgA8AC8AcwBjAHIAaQBwAHQAPgAKAAoAPABkAGkAdgAgAHsAQABhAHQAdABhAGMAaAAgAGYAbwBvAH0APgAKACAAIAB0AGUAcwBzAHQACgA8AC8AZABpAHYAPgA%3D&language=svelte

### Expected result

No lint rule should be triggered.

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by machadinhos** [comment]
svelte docs explaining the `{@attach }`
https://svelte.dev/docs/svelte/@attach
